drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
Ok, now I understand, what do you think about using forbidden in C/C++ symbols: $ or @ as a delimeter ?